install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -168,56 +321,89 @@ To enable writing to NFS volumes with SELinux enforcing on each node, run:
 # setsebool -P virt_use_nfs 1
 ----
 
-The `-P` option makes the bool persistent between reboots.
+The `-P` option above makes the bool persistent between reboots.
+
+The *virt_use_nfs* boolean is defined by the *_docker-selinux_* package. If an
+error is seen indicating that this bool is not defined, then ensure that this
+package has been installed.
 
 Additionally, in order to enable arbitrary container users to read and write the
-volume, each exported volume on the NFS server itself should conform to the
-following:
+volume, each exported volume on the NFS server should conform to the following:
 
 - Each export must be:
 +
 ----
 /<example_fs> *(rw,root_squash)
 ----
-- The firewall must be configured to allow traffic to the mount point, for NFSv4 the default port is 2049 (nfs).  
-  For NFSv3 there are three ports that need to be configured: 2049 (nfs), 20048 (mountd) and 111 (port mapper).
-+
-NFSv4:
```

Comment:
I saw your commit dropped this NFSv3 vs NFSv4 iptables rules stuff, but I suspect it was only accidental (it was added pretty recently in general). Unless you had a particular reason for dropping it?